README.md

```diff
@@ -78,14 +80,15 @@ var YourComponent = React.createClass({
 Displays the Activity View with actions relevant to the `shareObject` passed.
 
 __Arguments__
-- `shareObject` - An _Object_ containing one or more of the following keys `text`, `url`, `imageUrl` or `image`.
+- `shareObject` - An _Object_ containing one or more of the following keys `text`, `url`, `anchor`, `imageUrl` or `image`.
```

Comment:
We should specify that anchor is optional.  I'd rather the centered popup be the default behaviour.